omegaconf/base.py

```diff
@@ -205,10 +205,6 @@ class Container(Node):
 
     _metadata: ContainerMetadata
 
-    @abstractmethod
-    def pretty(self, resolve: bool = False, sort_keys: bool = False) -> str:
-        ...
```

Comment:
You have just pissed off every single use of OmegaConf :)

deprecate it, have it issue a warning and call OmegaConf.to_yaml().
See other examples in the codebase. (look for "warn").

Reply:
Oh shit true i'll undo that change asap

Reply:
No rush :), this is not landed and released yet.